schemas/message.ts

```diff
@@ -0,0 +1,29 @@
+import { EnvelopeIcon } from "@sanity/icons";
+import { defineField, defineType } from "sanity";
+
+const message = defineType({
+  name: "message",
+  title: "Mensaje",
+  type: "document",
+  // @ts-expect-error Wrong type
+  icon: EnvelopeIcon as unknown,
+  fields: [
+    defineField({
+      name: "name",
+      title: "*Nombre completo",
+      type: "string",
+    }),
+    defineField({
+      name: "email",
+      title: "*Email",
+      type: "string",
+    }),
```

Comment:
We can add some validations here, like so:

```suggestion
    defineField({
      name: "email",
      title: "*Email",
      type: "string",
      validation: (Rule) => Rule.required().email(),
    }),
```

Reply:
Good catch @TonyMckes! Should the message have max length or not?

Reply:
Not yet, maybe later!